ui/page/file/view.jsx

```diff
@@ -88,7 +102,18 @@ function FilePage(props: Props) {
     return () => {
       setPrimaryUri(null);
     };
-  }, [uri, hasFileInfo, fetchFileInfo, collectionId, clearPosition, fetchCostInfo, setViewed, setPrimaryUri]);
+  }, [
+    uri,
+    hasFileInfo,
+    fileInfo,
```

Comment:
I think fileInfo may be redundant here if we're passing hasFileInfo, but it's fine.